feature/account/server/settings/src/main/kotlin/app/k9mail/feature/account/server/settings/ui/incoming/content/IncomingFormItems.kt

```diff
@@ -53,6 +54,9 @@ internal fun LazyListScope.incomingFormItems(
             onTextChange = { onEvent(Event.ServerChanged(it)) },
             label = stringResource(id = R.string.account_server_settings_server_label),
             contentPadding = defaultItemPadding(),
+            keyboardOptions = KeyboardOptions(
+                autoCorrect = false,
+            ),
```

Comment:
There's no need for this to span three lines.

```suggestion
            keyboardOptions = KeyboardOptions(autoCorrect = false),
```

Reply:
Fixed